observability/monitoring/configuring-core-platform-monitoring/before-you-begin.adoc

```diff
@@ -6,7 +6,36 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-TBD
+The {product-title} installation program provides only a low number of configuration options before installation. Configuring most {product-title} framework components, including the cluster monitoring stack, happens after the installation.
 
+This section explains which monitoring components can be configured and how to prepare to configure the monitoring stack.
 
+[IMPORTANT]
+====
+* Not all configuration parameters for the monitoring stack are exposed.
+Only the parameters and fields listed in the xref:../../../observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#cluster-monitoring-operator-configuration-reference[Config map reference for the {cmo-full}] are supported for configuration.
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in xref:../../../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#scaling-cluster-monitoring-operator[Scaling the {cmo-full}] and verify that you have sufficient resources.
```

Comment:
Technically links to "Recommended infrastructure practices", which makes me think something might not be 100% correct with this xref.